packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -692,6 +703,7 @@ class ThemeData extends Diagnosticable {
 
   /// Linearly interpolate between two themes.
   ///
+  /// {@template flutter.material.themeData.lerp}
```

Comment:
NICE.

The block of text that follows appears about 40x in the flutter package, so we should probably extend this good work in a separate PR.

Reply:
Sounds good, I'll send a follow up PR replacing the rest of the usages in the flutter packages.